src/test/java/steppsDefinition/UbsPageStepsDefinition.java

```diff
@@ -0,0 +1,35 @@
+package steppsDefinition;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class UbsPageStepsDefinition {
+    WebDriver driver;
+
+    @Given("user is on UBS home page")
+    public void userIsOnUBSHomePage() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.get("https://www.ubs.com/ch/en.html");
+    }
+
+    @Then("web Element {string} is present on a page")
+    public void webElementIsPresentOnAPage(String arg0) {
+        driver.findElement(By.cssSelector(("button[id='headerLoginToggleButton']")));
```

Comment:
good start to Examples, even though you don't yet utilize them.
But I assume you already see the next step, how to utilize the `arg0` input parameter for a governed run.